### PR TITLE
Integrate with installer

### PR DESCRIPTION
**Pull request by ironcladlou**
Integrate with the component installer, and add a GCP installer shim to
facilitate local development. Add some docs.

### CODE REVIEW

File: playbooks/common/private/components.yml [hunk 1]
```diff
@@ -39,3 +39,6 @@
 
 - import_playbook: ../../openshift-management/private/config.yml
   when: openshift_management_install_management | default(false) | bool
+
+- import_playbook: ../../openshift-monitoring/private/config.yml
```
Comment (elad661) [inline playbooks/common/private/components.yml]: Is there a reason for calling this playbook twice? It's already called in line 34

Comment (ironcladlou) [reply playbooks/common/private/components.yml]: I didn't see that you had already done it; removed

**Comment by ironcladlou** [comment]
I can't get auth to work yet; my user is always rejected as invalid. Haven't debugged

**Comment by elad661** [comment]
@ironcladlou Auth worked for me when I tried deploying manually, haven't checked with the role yet. I'll debug this tomorrow.

**Pull request merged**